### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by anjaf**
This is the first draft of summarising the single cell MAGE-TAB fields for Expression Atlas processing. It contains empty templates for an IDF and an SDRF for plate-based experiments and one for droplet-based experiments.

### CODE REVIEW

File: IDF_template.txt [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+PubMed ID
+Publication DOI
+Publication Author List
+Publication Title
+Publication Status
+Publication Status Term Source REF
+Publication Status Term Accession Number
+
+Protocol Name
```
Comment (ngeorgeebi) [inline IDF_template.txt]: suggested we give examples of the protocols used and link to the relative EFO terms

File: README.md [hunk 2]
```diff
@@ -1,2 +1,24 @@
...
+[List of IDF fields](IDF_template.txt)
+
+### SDRF
+
+The SDRF (sample data relationship format) describes the samples, cDNA libraries ("extracts"), sequencing assays and their relation to each other. Due to distinct procedures for processing cells and analysing data, we use different formats and fields to describe plated-based experiments (such as Smart-seq2) and droplet-based experiments (such as 10x). For plate-based experiments, we capture single-cell specific metadata such as inferred cell type and single cell well quality. Here, each sample corresponds to a single cell (or well on the plate). For droplet-based experiments, we capture technology-specific metadata that describes the contruction of the library more closely (e.g. where barcodes are located). Here, one sample corresponds to a sequencing library that typically contains a few thousand cells. 
+
+Current templates:
+* [SDRF fields for plate-based experiments](SDRF_template_plate.txt)
+* [SDRF fields for droplet-based experiments](SDRF_template_droplet.txt)
+
```
Comment (ngeorgeebi) [inline README.md:19]: Here we should also provide links to the SDRFs we give as examples for each technology type in the paper
ie E-MTAB-xxxx.sdrf.txt

File: SDRF_template_droplet.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+Source Name	Comment[ENA_SAMPLE]	Comment[BioSD_SAMPLE]	Characteristics[organism]	Comment[cell count]	Comment[cell count pre filtering]	Comment[single cell quality] 	Material Type	Protocol REF	Extract Name	Comment[LIBRARY_LAYOUT]	Comment[LIBRARY_SELECTION]	Comment[LIBRARY_SOURCE]	Comment[LIBRARY_STRATEGY]	Comment[LIBRARY_STRAND]	Comment[NOMINAL_LENGTH]	Comment[NOMINAL_SDEV]	Comment[INSTRUMENT_MODEL]	Comment[single cell isolation]	Comment[library construction]	Comment[input molecule]	Comment[primer]	Comment[end bias]	Comment[spike in]	Comment[spike in dilution]	Comment[cdna read]	Comment[cdna read offset]	Comment[cdna read size]	Comment[cell barcode offset]	Comment[cell barcode read]	Comment[cell barcode size]	Comment[sample barcode offset]	Comment[sample barcode read]	Comment[sample barcode size]	Comment[umi barcode offset]	Comment[umi barcode read]	Comment[umi barcode size]	Protocol REF	Performer	Assay Name	Comment[technical replicate group]	Technology Type	Comment[ENA_EXPERIMENT]	Scan Name	Comment[ENA_RUN]	Comment[FASTQ_URI]	Comment[BAM_URI]	Comment[read1 file]	Comment[read2 file]	Comment[index1 file]	Comment[index2 file]
```
Comment (ngeorgeebi) [inline SDRF_template_droplet.txt:1]: would we need Comment[single cell quality] in droplet exps?

Comment (ngeorgeebi) [reply SDRF_template_droplet.txt:1]: we also might want to add Characteristics[inferred cell type] here as well - as we would like to include this in the SCEA (the SDRF becomes demultiplexed by cell)

**Review by ngeorgeebi — COMMENTED** [review]
please see comments in files

File: IDF_template.txt (hunk 1 above)
Comment (anjaf) [reply IDF_template.txt]: Added the mandatory protocol types to the IDF template

File: README.md (hunk 2 above)
Comment (anjaf) [reply README.md:19]: Yes. I've made an example folder and placed the two example experiments we have for 10x and Smart-seq2.

File: SDRF_template_droplet.txt (hunk 3 above)
Comment (anjaf) [reply SDRF_template_droplet.txt:1]: Yes, we are technically using the `Comment[single cell quality]` to filter out any runs that are labelled "not OK", even for 10x. I've seen one example where this was applicable (an experiment had some reads that had not been assigned to a sample). So leaving this in for now.

Comment (anjaf) [reply SDRF_template_droplet.txt:1]: Regarding the inferred cell type, this requires another solution in future. For now we cannot put this in the input SDRF.

**Pull request merged**